pydantic/_internal/_std_types_schema.py

```diff
@@ -90,47 +91,32 @@ def to_enum(input_value: Any, /) -> Enum:
         try:
             return enum_type(input_value)
         except ValueError:
-            # The type: ignore on the next line is to ignore the requirement of LiteralString
-            raise PydanticCustomError('enum', f'Input should be {expected}', {'expected': expected})  # type: ignore
+            raise PydanticCustomError('enum', 'Input should be {expected}', {'expected': expected})
```

Comment:
output is the same, but it's more correct to let pydantic-core take care of replacing the variables in the message template.